mito-ai/src/Extensions/AiChat/ChatTaskpane.tsx

```diff
@@ -1523,6 +1591,90 @@ const ChatTaskpane: React.FC<IChatTaskpaneProps> = ({
         return Array.isArray(item);
     };
 
+    const reviewAgentChanges = (): void => {
+        const currentNotebookSnapshot = getAIOptimizedCellsInNotebookPanel(agentTargetNotebookPanelRef.current);
+        currentNotebookSnapshotRef.current = currentNotebookSnapshot;
+
+        if (!notebookSnapshot || !currentNotebookSnapshot) {
+            return;
+        }
+
+        // Clear and populate the map of original cell states
+        cellStatesBeforeDiff.current.clear();
+
+        // Find cells that have changed between snapshots
+        const changedCells: { cellId: string, originalCode: string, currentCode: string }[] = [];
```

Comment:
Can we create a type for `{ cellId: string, originalCode: string, currentCode: string }` and use that across the codebase? Right now we have to type this out in a bunch of places